src/govuk/settings/_links.scss

```diff
@@ -4,43 +4,62 @@
 
 /// Enable new link styles
 ///
+/// If enabled, the link styles will change. Underlines will:
+///
+/// - be consistently thinner and a bit further away from the link text
+/// - have a clearer hover state, where the underline gets thicker to make the
+///   link stand out to users
+///
+/// These changes are an opt-in feature because you may:
+///
+/// - need to make changes to your service to introduce these new link styles
+///   consistently
+/// - want to avoid opting in if you have links within a multiple-column layout
+///   in your CSS - there is [a Chromium bug that affects such
+///   links](https://github.com/alphagov/govuk-frontend/issues/2204)
+///
+/// These new link styles will be enabled by default in a future major version
```

Comment:
It's probably best to avoid promising this - you never know what might happen. It's also probably not useful to a developer using the documentation right now (unless you're expecting developers to write their code to accommodate the potential change) - it's better to communicate this in a release note at the time.

Reply:
@m-green Yeah, we're communicating this in the upcoming release notes.

@36degrees I'll remove the text here about what we're doing in future, and I'll make a note to update these Sass docs when we're enabling link styles by default.